client/luigi-client.js

```diff
@@ -196,10 +198,10 @@
           var navigationOpenMsg = {
             msg: 'luigi.navigation.open',
             sessionId: sessionId,
-            params: Object.assign(
-              { link: path, relative: relativePath },
-              options
-            )
+            params: Object.assign(options, {
```

Comment:
You know, that this updates the options object, is this intentionally? Previously it was a new object that receives additional options.

Reply:
We are mutating this object in other places, so this way we are making sure the content of this object is always up to date. Additionally, as discussed offline the object is generated with default values every time we use the link manager